### PR TITLE
User profiles

### PR DESCRIPTION
**Pull request by Zekereyez**
Added user profile feature with the ability to query a users profile information and load into profile view controller.

### CODE REVIEW

File: Squablr/Assets.xcassets/29.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (rhammoud0418) [inline Squablr/Assets.xcassets/29.imageset/Contents.json:1]: Again, this PR should just be focused on user profiles, not editing. So this icon you added would be in a different PR.

File: Squablr/View Controllers/EditProfileViewController.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//
```
Comment (rhammoud0418) [inline Squablr/View Controllers/EditProfileViewController.h:1]: Again, this PR should just be focused on user profiles, not editing. So this code would go in another PR.

File: Squablr/View Controllers/ProfileViewController.m [hunk 3]
```diff
@@ -18,21 +19,59 @@ @implementation ProfileViewController
...
             UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
             LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"loginVC"];
             self.view.window.rootViewController = loginViewController;
         }
     }];
 }
 
--(void) welcome {
-    _welcomeMessage.text = [NSString stringWithFormat:@"%@%@%@", @"Hello, ", [PFUser currentUser].username, @"!"];
+-(void) queryUserProfileInfo {
```
Comment (rhammoud0418) [inline Squablr/View Controllers/ProfileViewController.m:40]: Think it might be a good idea to start putting all Parse query code in its own file. Remind me to discuss this with you, we can probably do this in a follow up PR.

File: Squablr/View Controllers/ProfileViewController.m [hunk 4]
```diff
@@ -18,21 +19,59 @@ @implementation ProfileViewController
...
+        }
+        else {
+            // Log error
+            NSLog(@"%@", error.localizedDescription);
+        }
+    }];
+}
+
+-(void) loadUserProfileInfo {
+    // Extract the array element properties for the user and assign into profile labels
```
Comment (rhammoud0418) [inline Squablr/View Controllers/ProfileViewController.m:64]: As a sanity check I would make sure that arrayOfUserInfo has exactly 1 entry just to be extra safe.

**Review by rhammoud0418 — COMMENTED** [review]
Address comments, otherwise LGTM!

File: Squablr/View Controllers/ProfileViewController.m (hunk 3 above)
Comment (Zekereyez) [reply Squablr/View Controllers/ProfileViewController.m:40]: Will do
